strapi/strapi
# fix(Blocks): remove pop up link and reopening on save/publish

### What does it do?

- Removing old pop-up with edit/delete options.
- Replacing this pop-up with existing save/cancel pop-up, with an addition of 'Remove' button.
- Focus link URL on opening link pop-up.
- Validation check added on save, so user has to enter a valid URL [reference](https://www.figma.com/file/MWPKHNT98LLWHGRp3MyWqu?node-id=1339:17473&mode=design#643292761)
- Fixed issue of reopening pop-up on save/publish or copy-paste a link, [Issue](https://strapi-inc.atlassian.net/browse/CS-391)

### How to test it?
[Designs](https://strapi-inc.atlassian.net/browse/CS-401)
1. Check by adding a new link by clicking a toolbar link option.
2. Check by copy-paste a link
3. Clicking save/publish a content should not open link pop-up.
